lib/rubocop/node_pattern.rb

```diff
@@ -577,6 +577,11 @@ def marshal_dump
       pattern
     end
 
+    def ==(other)
+      other.is_a?(NodePattern) && other.pattern == pattern
```

Comment:
Node patterns allow arbitrary use of whitespace without changing semantics. We might want to strip out all spaces before comparing them. 🙂

Reply:
Good point :+1:

Stripping all spaces would be too permissive though, as `"12 34"` and `"1 234"` aren't the same. I've reused the right code so it should deal correctly with all whitespace now.